### PR TITLE
Update staff page to reflect operations staff

### PR DESCRIPTION
**Pull request by hsugden**
image

### CODE REVIEW

File: resources/views/site/staff.blade.php [hunk 1]
```diff
@@ -72,11 +72,12 @@ class="img-responsive center-block profile-picture"/>
...
                                 <td>Aerodrome Operations Coordinator</td>
-                                <td>Vacant</td>
+                                <td>Chad Byworth</td>
                             </tr>
                             <tr>
-                                <td>Sector File Coordinator</td>
-                                <td>Vacant</td>
+                                <td>Sector File Coordinators</td>
+                                <td>Luke Brown<br>
+								Thomas Mills</td>
```
Comment (CalumTowers) [inline resources/views/site/staff.blade.php:80]:
```html
<tr>
    <td>Sector File Coordinator</td>
    <td>Luke Brown</td>
</tr>
<tr>
    <td>Sector File Coordinator</td>
    <td>Thomas Mills</td>
</tr>
```
Each person should have their own row to keep it consistent with other areas of this page.

**Review by CalumTowers — CHANGES_REQUESTED** [review]
Thanks for the PR 👍

File: resources/views/site/staff.blade.php (hunk 1 above)
Comment (hsugden) [reply resources/views/site/staff.blade.php]: Fixed!

**Review by CalumTowers — APPROVED** [review]
👍

**Pull request merged**